### DISCUSSION TITLE
V9: Is there a way to customize tooltip text for an AvatarGroup?

### DISCUSSION THREAD

**Discussion by emiliechung** [microsoft/fluentui#27696] [Q&A] [answered]
The tooltip text for overflowing items for an AvatarGroup is "View more people.". Is there a way to customize the tooltip text? I need to translate the text to different languages :thinking: 

![image](https://user-images.githubusercontent.com/16834516/234534580-d3d9bd14-0c23-4eb3-9121-8f3c21996ccb.png)

There is an old bug report that is related to this. Unfortunately, it was closed due to inactivity, https://github.com/microsoft/fluentui/issues/23038 :disappointed:

**Comment by layershifter** [comment, marked as answer]
@emiliechung you can pass `tooltip` to `AvatarGroupPopover`:

```tsx
<AvatarGroupPopover
  indicator="count"
  tooltip={{ content: "Custom", relationship: "label" }}
/>
```

https://codesandbox.io/s/vigorous-lumiere-lu3rgo?file=/example.tsx:690-812

**Reply by lekoaf** [reply]
Can this be found anywhere on the documentation page? Couldn't find anything.

Would you like me to add a PR for it? @layershifter.

**Reply by layershifter** [reply]
@lekoaf it's great idea to have an example for it, feel free to open a PR 👍

**Comment by emiliechung** [comment]
Thanks! It works :smile:
